de

```diff
@@ -19,12 +19,8 @@ function fail {
     exit 1
 }
 
-if [ ! -f ~/.de ]
-then
-    fail
-else:
-    source ~/.de
-fi
+if [ ! -f ~/.de ]; then fail; else source ~/.de; fi
```

Comment:
I didn't mean to put it all on the same line, just the `if` and the `then`.

Also, say `fail "Config not found!"` or something like that rather than just failing.